.analysis_options

```diff
@@ -0,0 +1,6 @@
+# Analyzer options file.
+
+analyzer:
+  exclude:
+    - test_package_bad/
```

Comment:
I can add if you think it's be necessary; I don't _think_ they are required in .gitignore files (https://github.com/dart-lang/dartdoc/blob/master/.gitignore#L13).